hnn_core/network.py

```diff
@@ -349,10 +350,16 @@ def add_evoked_drive(self, name, *, mu, sigma, numspikes,
             Standard deviation of event time distribution
         numspikes : int
             Number of spikes at each target cell
-        sync_within_trial : bool
-            If True, the target cells receive each numspike synchronously. By
-            default (False), spike times arriving at each target cell are
-            sampled independently using the Gaussian parameteres (mu, sigma).
+        n_drive_cells : int | 'n_cells'
+            The number of drive cells (i.e., ArtificialCell objects) that each
```

Comment:
can we leave out "ArtificialCell" in the description? I am not sure we want to expose it in the public API yet